frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -398,14 +459,57 @@ export default function OntologyDagView({
   const yMax = height - defaultMargin.top - defaultMargin.bottom;
   const xMax = width - defaultMargin.left - defaultMargin.right;
 
+  const organismText =
+    formattedSelectedOrganism === "homo_sapiens" ? "human" : "mouse";
```

Comment:
For context and readability - maybe worth adding a comment on `formattedSelectedOrganism` that the current possible values are `"homo_sapiens" and "mus_musculus". Or perhaps better yet, create a constant to map between value to displayed text like so:

```ts
/**
 * The keys are expected to match the pattern created by `formattedSelectedOrganism` above
 */
const SELECTED_ORGANISM_TO_DISPLAY_TEXT = {
  homo_sapiens: "human",
  mus_musculus: "mouse",
}

organismText = SELECTED_ORGANISM_TO_DISPLAY_TEXT[formattedSelectedOrganism] || "unknown"
```